web/src/enterprise/codeintel/CodeIntelIndexPage.tsx

```diff
@@ -198,24 +197,19 @@ export const CodeIntelIndexPage: FunctionComponent<Props> = ({
                         </tbody>
                     </table>
 
-                    <div className="action-container">
-                        <div className="action-container__row">
-                            <div className="action-container__description">
-                                <h4 className="action-container__title">Delete this index</h4>
-                                <div>Deleting this index will remove it from the index queue.</div>
-                            </div>
-                            <div className="action-container__btn-container">
-                                <button
-                                    type="button"
-                                    className="btn btn-danger action-container__btn"
-                                    onClick={deleteIndex}
-                                    disabled={deletionOrError === 'loading'}
-                                    data-tooltip="Delete index"
-                                >
-                                    <DeleteIcon className="icon-inline" />
-                                </button>
-                            </div>
-                        </div>
+                    <div className="mt-4 p-2 pt-2">
```

Comment:
`p-2` also sets the `padding-top` :)
```suggestion
                    <div className="mt-4 p-2">
```

Reply:
Duhh 🤦 I had different pt before I adjusted it.